examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1672 @@
+{
```

Comment:
<p>MAYBE: you could include <code>sy.logger.remove()  to remove the memory log statements at output.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>